localstack/services/internal.py

```diff
@@ -270,45 +271,42 @@ class LocalstackResources(Router):
     """
 
     def __init__(self):
-        super().__init__(dispatcher=resource_dispatcher(pass_response=False))
+        super().__init__(dispatcher=handler_dispatcher())
```

Comment:
q: One of the main motivations for the `LocalstackResources` was to explicitly encapsulate all these internal APIs in a single "namespace" / path part (`/_localstack`). This is somewhat lost with this change. I still makes sense to have these routes served by a different handler / router than the other - service specific - routes, but it feels like this class is losing it's purpose / concern. What's your take on this?

Reply:
the main purpose of the separation is still this here: https://github.com/localstack/localstack/blob/4aaa41a178edd5eb9d1e3ba10ba52ac62730732e/localstack/aws/handlers/internal.py#L28-L38

but i agree that technically the `LocalstackResource` Router subclass is no longer needed. Instead, the `add_default_routes` should be something that's done as a plugin somewhere, and `get_internal_apis` just becomes a well-known `Router` instance, like the edge `ROUTER`. Let's do that in a follow-up PR :+1: